myglayapp/main.go

```diff
@@ -0,0 +1,416 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"log/slog"
+	"runtime"
+	"sort"
+	"strings"
+
+	"github.com/go-gl/gl/v3.3-core/gl"
+	"github.com/go-gl/glfw/v3.3/glfw"
+	"github.com/go-gl/mathgl/mgl32"
+	"github.com/soypat/glay"
```

Comment:
I want to avoid adding dependencies to glgl, especially multi-million line C libraries. Let's add it under an `examples` directory:
- `examples/_gl-renderer`: The leading underscore will omit the directory from the glay module. Inside of this excluded directory run `go mod init gl-renderer` and bam, you have a module within a module that need not share dependencies.
![image](https://github.com/user-attachments/assets/4aa79d5b-10b1-4007-b67c-2d41c14a222d)